CMakeLists.txt

```diff
@@ -49,16 +49,18 @@ os_target_config()
 global_target_config()
 option(BUILD_CV_KINFU_EXAMPLE "Build OpenCV KinectFusion example" OFF)
 
+include(CMake/install_config.cmake)
```

Comment:
@radfordi , can you split Cmake into a separate PR ? I recognize potential conflicts with PR4073 that is scheduled as well

Reply:
Done.  See #4094.